sidecar/config/flags.go

```diff
@@ -36,6 +36,7 @@ const (
 	controllerTokenFlag = "controller_token"
 	controllerPollFlag  = "controller_poll"
 	superviseFlag       = "supervise"
+	healthChecksFlag    = "health_checks"
```

Comment:
Flag name (`health_checks`) is different than the corresponding YAML key (`healthchecks`).
For consistency, these should be the same. I personally prefer `healthchecks` as it reads cleaner.